src/transformers/core_model_loading.py

```
@@ -124,8 +151,15 @@ class ConversionOps:
 
     @abstractmethod
     def convert(
-        self, value: Union[dict[str, torch.Tensor], Sequence[torch.Tensor], torch.Tensor], *args, **kwargs
-    ) -> torch.Tensor:
+        self,
+        value: dict[str, Any],
+        *,
```

Having the wildcard `*` here is very error prone IMO -> would just lead to silently skip args